spec/lib/govuk_publishing_components/presenters/schema_org_spec.rb

```diff
@@ -75,7 +75,7 @@
 
       expect(structured_data['@type']).to eql("Dataset")
       expect(structured_data['name']).to eql("Dataset Title")
-      expect(structured_data['description']).to eql("Dataset description")
+      expect(structured_data['description']).to eql("Dataset body" + ("a" * 4988))
```

Comment:
I think it would be better instead to check that the length is 5000.

Reply:
done, now checks the length of the description to check that it's only 5000 chars long